Makefile.am

```diff
@@ -142,14 +142,14 @@ fftw3@PREC_SUFFIX@.pc: fftw.pc
 pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = fftw3@PREC_SUFFIX@.pc
 
-FindFFTW3@PREC_SUFFIX@.cmake: $(top_srcdir)/FindFFTW3.cmake.in
+FFTW3@PREC_SUFFIX@Config.cmake: $(top_srcdir)/FFTW3Config.cmake.in
 	$(SED) \
 		-e 's|[@]PREC_SUFFIX@|@PREC_SUFFIX@|g' \
 		-e 's|[@]libdir@|$(libdir)|g' \
 		-e 's|[@]includedir@|$(includedir)|g' \
-		$(top_srcdir)/FindFFTW3.cmake.in > $@
-cmakedir = $(datarootdir)/cmake/Modules
-cmake_DATA = FindFFTW3@PREC_SUFFIX@.cmake
+		$(top_srcdir)/FFTW3Config.cmake.in > $@
+cmakedir = $(libdir)/cmake/fftw3
+cmake_DATA = FFTW3@PREC_SUFFIX@Config.cmake
```

Comment:
I think `FFTW3@PREC_SUFFIX@ConfigVersion.cmake.in` is missing here and the related make rule.